### PR TITLE
Misc integration updates to simplify integration with other projects

### PR DESCRIPTION
**Pull request by gabog**
## Description
This PR contains a series of updates that resulted from trying to integrate the skills with other projects.

Here is a summary of the changes: 

- Added id attribute to adaptive card jsons (this is required by some clients for rendering).
- Updated some responses in CalendarSharedResponses to improve UX.
- Updated inputHint in CreateEventResponses.json (shouldn't open the mic at the end of the flow).
- Updated inputHint in SummaryResponse.json (shouldn't open the mic at the end of the flow).
- Removed references to ContentModerator from the skills (this is not used by the skills).
- Disabled StyleCop rules SA1413 and SA1117 as discussed with @lzc850612 and @lauren-mills.

### CODE REVIEW

File: solutions/Virtual-Assistant/src/csharp/VirtualAssistant.ruleset [hunk 1]
```diff
@@ -16,10 +16,12 @@
     <Rule Id="SA1011" Action="None" />
     <Rule Id="SA1200" Action="None" />
     <Rule Id="SA1101" Action="None" />
+    <Rule Id="SA1117" Action="None" />
     <Rule Id="SA1129" Action="None" />
     <Rule Id="SA1305" Action="Warning" />
     <Rule Id="SA1309" Action="None" />
     <Rule Id="SA1412" Action="Warning" />
+	<Rule Id="SA1413" Action="None" />
```
Comment (lzc850612) [inline solutions/Virtual-Assistant/src/csharp/VirtualAssistant.ruleset]: extra spaces

Comment (gabog) [reply solutions/Virtual-Assistant/src/csharp/VirtualAssistant.ruleset]: Updated.

File: solutions/Virtual-Assistant/src/csharp/skills/calendarskill/Dialogs/Shared/Resources/CalendarSharedResponses.json [hunk 2]
```diff
@@ -42,18 +42,6 @@
...
-        "text": "Sure, we can do this later.",
-        "speak": "Sure, we can do this later."
-      },
-      {
-        "text": "Sure, we can start this later.",
-        "speak": "Sure, we can start this later."
-      },
-      {
-        "text": "No problem, you can try again at another time.",
-        "speak": "No problem, you can try again at another time."
```
Comment (lzc850612) [inline solutions/Virtual-Assistant/src/csharp/skills/calendarskill/Dialogs/Shared/Resources/CalendarSharedResponses.json:55]: why remove these?

File: solutions/Virtual-Assistant/src/csharp/skills/calendarskill/Dialogs/Summary/Resources/SummaryResponses.json [hunk 3]
```diff
@@ -29,8 +29,8 @@
     "ShowOneMeetingSummaryMessage": {
         "replies": [
             {
-                "text": "I found 1 event for you:",
-                "speak": "I found 1 event for you: It is {EventName1} for {EventDuration}."
+        "text": "I found only one event for you today:",
+        "speak": "I found only one event for you today: {EventName1} for {EventDuration}."
```
Comment (lzc850612) [inline solutions/Virtual-Assistant/src/csharp/skills/calendarskill/Dialogs/Summary/Resources/SummaryResponses.json]: spacing problem

File: solutions/Virtual-Assistant/src/csharp/skills/calendarskill/Dialogs/Summary/Resources/SummaryResponses.json [hunk 4]
```diff
@@ -88,8 +88,8 @@
     "ReadOutMorePrompt": {
         "replies": [
             {
-                "text": "Do you want to hear more? Which one do you want to hear?",
-                "speak": "Do you want to hear more? Which one do you want to hear?"
+        "text": "Do you want to hear more appointments?",
+        "speak": "Do you want to hear more appointments?"
```
Comment (lzc850612) [inline solutions/Virtual-Assistant/src/csharp/skills/calendarskill/Dialogs/Summary/Resources/SummaryResponses.json]: spacing problem

**Review by lzc850612 — CHANGES_REQUESTED** [review]
left comments

File: solutions/Virtual-Assistant/src/csharp/skills/calendarskill/Dialogs/Shared/Resources/CalendarSharedResponses.json (hunk 2 above)
Comment (gabog) [reply solutions/Virtual-Assistant/src/csharp/skills/calendarskill/Dialogs/Shared/Resources/CalendarSharedResponses.json:55]: they don't really make sense in the conversation, the user is not "trying" anything, it is just setting up an appointment.

File: solutions/Virtual-Assistant/src/csharp/skills/calendarskill/Dialogs/Summary/Resources/SummaryResponses.json (hunk 3 above)
Comment (gabog) [reply solutions/Virtual-Assistant/src/csharp/skills/calendarskill/Dialogs/Summary/Resources/SummaryResponses.json]: Updated

File: solutions/Virtual-Assistant/src/csharp/skills/calendarskill/Dialogs/Summary/Resources/SummaryResponses.json (hunk 4 above)
Comment (gabog) [reply solutions/Virtual-Assistant/src/csharp/skills/calendarskill/Dialogs/Summary/Resources/SummaryResponses.json]: Updated

**Review by lzc850612 — APPROVED** [review]

**Pull request merged**